### ISSUE TITLE
Add support for Swagger

### ISSUE THREAD

**Issue by phillip-kruger** [smallrye/smallrye-open-api#752]
https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Annotations

**Comment by robotmrv** [comment]
Hi @phillip-kruger   
What is the status of this issue?  
Are there any plans/timelines for it?

**Comment by phillip-kruger** [comment]
At this point it's just a wishlist. Are you keen to contribute?

**Comment by matheusandre1** [comment]
@phillip-kruger Can I submit a pull request for this?

**Comment by MikeEdgar** [comment]
Hi @matheusandre1 do you have a working draft for this? If so, I'm curious how much new code is involved. My concern with this issue is that we will effectively double the size of the scanner code base and either maintenance becomes too burdensome or the Swagger part drifts out of date.

**Comment by matheusandre1** [comment]
> [re1](https://github.com/matheusandre1) do you have a working draft for this? If so, I'm curious how much new code is involved. My concern with this issue is that we will effectively double the size of the scanner code base and either maintenance becomes too burdensome or the Swagger part drifts out of date.

hi @MikeEdgar, I don't have a draft of this; I would explore the codebase first. This project is important for the `quarkus-openapi-generator`, to which I've been contributing, and I want to be able to contribute to that project as well.

**Comment by MikeEdgar** [comment]
Ok, thanks. Is the openapi-generator using Swagger annotations somehow? I didn't see that.

**Comment by matheusandre1** [comment]
> Ok, thanks. Is the openapi-generator using Swagger annotations somehow? I didn't see that.

No, I was talking about the `smallrye-open-api` project itself; I'll look for another open issue to contribute to.
